packages/docusaurus-theme-common/src/utils/useThemeConfig.ts

```diff
@@ -32,11 +34,39 @@ export type Navbar = {
   logo?: NavbarLogo;
 };
 
+export type ColorModeConfig = {
```

Comment:
FYI @armano2 this is likely to conflict with your work here: https://github.com/facebook/docusaurus/pull/4433

Reply:
no worries, i have to redo this pr anyway, and i'm going to split it to smaller pieces